src/modules/Extension/Service.php

```diff
@@ -389,19 +389,53 @@ public function deactivate(\Model_Extension $ext)
         return true;
     }
 
-    public function uninstall(\Model_Extension $ext)
+    /**
+     * @throws \Box_Exception
+     */
+    public function uninstall(\Model_Extension $ext): bool
     {
         $this->deactivate($ext);
 
-        switch ($ext->type) {
+        $this->di['db']->trash($ext);
```

Comment:
This is unneeded, `deactivate` already does this:
https://github.com/FOSSBilling/FOSSBilling/blob/0fcac8d5c2d9da5d0750f348075a709b6c47bde3/src/modules/Extension/Service.php#L387